chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -415,10 +433,15 @@ bool BraveRenderViewContextMenu::IsCommandIdEnabled(int id) const {
     case IDC_AI_CHAT_CONTEXT_REWRITE:
       return ai_rewriter::features::IsAIRewriterEnabled();
 #endif
+
+    case IDC_CONTENT_CONTEXT_OPENLINK_SPLIT_VIEW:
+      return CanOpenSplitViewForWebContents(source_web_contents_->GetWeakPtr());
+
```

Comment:
nit: I think we can remove whitespace around lines

```suggestion
    case IDC_CONTENT_CONTEXT_OPENLINK_SPLIT_VIEW:
      return CanOpenSplitViewForWebContents(source_web_contents_->GetWeakPtr());
```

Reply:
Removed.